help/destinations/catalog/advertising/amazon-ads.md

```diff
@@ -141,6 +170,7 @@ This section captures the functionality and significant documentation updates ma
 
 |Release month|Update type|Description|
 |---|---|---|
+|January 2024|Functionality and documentation update| Added new documentation for AMC.
```

Comment:
Perhaps we can enhance this part by saying: 

Added the option to export audiences to be used in Amazon Marketing Cloud (AMC). There's also a missing pipe symbol after the colon. it's needed so the table renders properly.